syntax = "proto3";

service PythonService {
  rpc SendCode(PythonRequest) returns (PythonResponse) {}
}

message PythonRequest {
  string script = 1;
}

message PythonResponse {
  string result = 1;
}